providers/celery/tests/provider_tests/celery/executors/test_celery_executor.py

```diff
@@ -147,43 +147,6 @@ def test_gauge_executor_metrics(self, mock_stats_gauge, mock_trigger_tasks, mock
         ]
         mock_stats_gauge.assert_has_calls(calls)
 
-    @pytest.mark.parametrize(
-        "command, raise_exception",
-        [
-            pytest.param(["true"], True, id="wrong-command"),
-            pytest.param(["airflow", "tasks"], True, id="incomplete-command"),
-            pytest.param(["airflow", "tasks", "run"], False, id="complete-command"),
-        ],
-    )
-    def test_command_validation(self, command, raise_exception):
```

Comment:
This test is gone as we don't need to worry about this anymore

Reply:
Won't we want it for the old core provider compat tests? Maybe should be kept but put behind a skipif(AF3...)?

Reply:
+1 on what Jed has to say, we will still continue to run the compat tests